python/hail/dataset.py

```diff
@@ -2360,7 +2360,8 @@ def ld_prune(self, r2=0.2, window=1000000, memory_per_core=256, num_cores=1):
 
     @handle_py4j
     @requireTGenotype
-    def ld_matrix(self):
+    @typecheck_method(compute_locally=nullable(bool))
```

Comment:
The reason it is this way is because there are really three choices. True, False, or None. None indicates that Hail will choose what is best, True or False indicate a particular user preference.